diff -ur NAMD_2.8_Source.orig/Makefile NAMD_2.8_Source/Makefile
--- NAMD_2.8_Source.orig/Makefile	2011-07-30 13:17:02.513705293 -0500
+++ NAMD_2.8_Source/Makefile	2011-07-30 13:18:03.363943762 -0500
@@ -362,7 +362,7 @@
 
 namd2:	$(INCDIR) $(DSTDIR) $(OBJS) $(LIBS)
 	$(MAKEBUILDINFO)
-	$(CHARMC) -verbose -ld++-option \
+	$(CHARMC) $(LDFLAGS) -verbose -ld++-option \
 	"$(COPTI)$(CHARMINC) $(COPTI)$(INCDIR) $(COPTI)$(SRCDIR) $(CXXOPTS)" \
 	-module NeighborLB -module HybridLB -module RefineLB -module GreedyLB -language charm++ \
 	$(BUILDINFO).o \
@@ -404,7 +404,7 @@
 	$(COPY) $(CHARM)/bin/charmd_faceless.exe charmd_faceless.exe
 
 psfgen:	$(DSTDIR) $(SBOBJS) $(PLUGINOBJS)
-	$(CC) $(SBCFLAGS) -o psfgen $(SBOBJS) $(PLUGINOBJS) $(TCLLIB) $(TCLAPPLIB) -lm
+	$(CC) $(SBCFLAGS) $(LDFLAGS) -o psfgen $(SBOBJS) $(PLUGINOBJS) $(TCLLIB) $(TCLAPPLIB) -lm
 
 psfgen.exe:	$(DSTDIR) $(SBOBJS) $(PLUGINOBJS) $(TCLDLL)
 	$(CC) $(SBCFLAGS) -o psfgen $(SBOBJS) $(PLUGINOBJS) $(TCLLIB) $(TCLAPPLIB) -lm
@@ -413,17 +413,17 @@
 	$(CC) $(CFLAGS) -o diffbinpdb $(SRCDIR)/diffbinpdb.c -lm
 
 flipdcd:	$(SRCDIR)/flipdcd.c
-	$(CC) $(CFLAGS) -o $@ $(SRCDIR)/flipdcd.c || \
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(SRCDIR)/flipdcd.c || \
 	echo "#!/bin/sh\necho unavailable on this platform" > $@; \
 	chmod +x $@
 
 flipbinpdb:	$(SRCDIR)/flipbinpdb.c
-	$(CC) $(CFLAGS) -o $@ $(SRCDIR)/flipbinpdb.c || \
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(SRCDIR)/flipbinpdb.c || \
 	echo "#!/bin/sh\necho unavailable on this platform" > $@; \
 	chmod +x $@
 
 fixdcd:	$(SRCDIR)/fixdcd.c
-	$(CC) $(CFLAGS) -o fixdcd $(SRCDIR)/fixdcd.c
+	$(CC) $(CFLAGS) $(LDFLAGS) -o fixdcd $(SRCDIR)/fixdcd.c
 
 dumpdcd:	$(SRCDIR)/dumpdcd.c
 	$(CC) $(CFLAGS) -o dumpdcd $(SRCDIR)/dumpdcd.c
