From 8506081540bf289a5283f15f519c082e3d976133 Mon Sep 17 00:00:00 2001
From: Nicolas Bigaouette <nbigaouette@gmail.com>
Date: Mon, 12 Mar 2012 12:06:42 -0400
Subject: [PATCH] Use system-wide dsfmt, suitesparse-wrapper and gmp-wrapper

---
 external/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/external/Makefile b/external/Makefile
index ba37cb8..f724779 100644
--- a/external/Makefile
+++ b/external/Makefile
@@ -22,7 +22,7 @@ include $(JULIAHOME)/Make.inc
 
 OS = $(shell uname)
 ARCH = $(shell uname -m)
-LIBS = fdlibm dsfmt double-conversion amos suitesparse-wrapper gmp-wrapper
+LIBS = fdlibm double-conversion amos dsfmt
 
 ifeq ($(OS), Linux)
 LIBS += unwind
-- 
1.7.9.48.g85da4d.dirty

